tests/e2e/integration/wallets.spec.js

```diff
@@ -0,0 +1,323 @@
+const testKeyPath = 'test@notary.priv'
+
+const TOKEN = 'BasicAttentionToken'
+
+describe('Test wallets page', () => {
+  it('Make auth', () => {
+    cy.visit('/')
+    cy.login(testKeyPath)
+    cy.server()
+    cy.route('GET', '/data/histoday*limit=30', 'fixture:crypto-api/histoday30.json')
+    cy.route('GET', '/data/histoday*limit=365', 'fixture:crypto-api/histoday365.json').as('getHistoday365')
+    cy.wait('@getHistoday365')
+  })
+
+  it('Go to wallets page', () => {
+    cy.get('li.el-menu-item:nth-of-type(2)').click({ force: true })
+    cy.url().should('contain', 'wallets/')
+  })
+
+  it('Search for waller', () => {
+    cy.get('.el-input__inner')
```

Comment:
The same applies to other selectors: I think we should use IDs or special selectors like `:contains()` more for readability and maintainability, for I feel that `cy.get("#some-search-field")` is preferable to `cy.get(".el-input__inner")` in that:

* it is obvious that which element this is
* it is more semantic
* it is resistant to change of HTML (it is true to :nth-child especially)

Actually, I used an ID and `:contains` selector in my PR.
https://github.com/d3ledger/back-office/pull/33/files#diff-b90a838ff70d753d84eedf2f7eee7959R55

But I'm not sure how strictly we should apply this rule. 🤔 I'd like to hear your opinion.